src/ToricVarieties/LineBundles/attributes.jl

```diff
@@ -0,0 +1,25 @@
+#####################
+# 1. Defining data of a line bundle
+#####################
+
+@doc Markdown.doc"""
+    divisor_class(l::ToricLineBundle)
+
+Returns the divisor class which defines the toric line bundle `l`.
+```
```

Comment:
```suggestion
```
I don't think this is needed. See also the other docstrings.

Reply:
You are right. My bad. Fixed.